pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -381,6 +381,24 @@ type CommonPrometheusFields struct {
 	// TracingConfig configures tracing in Prometheus. This is an experimental feature, it may change in any upcoming release in a breaking way.
 	// +optional
 	TracingConfig *PrometheusTracingConfig `json:"tracingConfig,omitempty"`
+	// BodySizeLimit defines per-scrape on response body size.
+	// Only valid in Prometheus versions 2.45.0 and newer.
```

Comment:
Can you add `// +optional` to this one and the following ones?

Reply:
Hi @mcbenjemaa, asking just to be sure... have you noticed xiu's comments?

It should be good to merge after addressing it :)

Reply:
YES SURE

Reply:
done